dist/index.html

```diff
@@ -1,21 +1 @@
-<!DOCTYPE html>
-<html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>To do list</title>
-<script defer src="main.js"></script></head>
-
-<body>
-    <section class="to-do-list">
-        <div class="title">
-            <h2>Today's to do</h2>
-        </div>
-        <input type="text" class="input-text" placeholder="...">
-        <div class="the-list"></div>
```

Comment:
- [ ] I think you should have a "Clear all completed" button below this element. In this milestone, you are also required to be able to delete all the completed tasks in one button click. :+1: Please look back to the project's requirements [here](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md).